app/src/main/java/com/example/kuit6_week1/MainActivity.kt

```diff
@@ -19,29 +37,208 @@ class MainActivity : ComponentActivity() {
+                    .background(Color.Black)
+            )
+            Box(
+                modifier = modifier
+                    .width(60.dp)
+                    .height(60.dp)
+                    .clip(CircleShape)
+                    .background(Color.Black)
+            )
+        }
+        SampleRow(modifier = Modifier.padding(top = 20.dp)){
+            Box(
+                modifier = modifier
+                    .size(60.dp)
+                    .border(
+                        color = Color.Black,
+                        width = 2.dp,
+                        shape = RoundedCornerShape(20.dp)
+                    )
+
+            )
+            Box(
+                modifier = modifier
+                    .size(60.dp)
+                    .border(
+                        color = Color.Black,
+                        width = 3.dp,
+                        shape = CircleShape
+                    )
+
+            )
+        }
+        SampleRow (
+            modifier = Modifier.padding(top = 20.dp)
+        ) {
+            Box(
+                modifier = modifier
+                    .size(76.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black
+                    )
+                    .padding(8.dp)
+                    .background(Color.Red)
+            )
+            Box(
+                modifier = modifier
+                    .size(76.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black
+                    )
+                    .padding(8.dp)
+                    .background(Color.Blue)
+                    .clip(CircleShape)
+            )
+        }
+        SampleRow (
+            modifier = Modifier.padding(20.dp)
+        ){
+            Image(
+                painter = painterResource( R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier.size(80.dp)
+            )
+
+            Image(
+                painter = painterResource(R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier
+                    .size(80.dp)
+                    .clip(CircleShape)
+            )
+
+        }
+        Text(
+            modifier = Modifier
+                .padding(top = 20.dp)
+                .fillMaxWidth()
+                .border(2.dp, color = Color.Black)
+                .padding(10.dp),
+
+            text = "Hello World!",
+            fontSize = 20.sp,
+            fontStyle = FontStyle.Normal,
+            fontWeight = FontWeight.Bold,
+            color = Color.Black,
+            textAlign = TextAlign.Start
+        )
+    }
+}
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-    Text(
-        text = "Hello $name!",
-        modifier = modifier
-    )
+fun Profile_Card(modifier: Modifier = Modifier) {
```

Comment:
```suggestion
fun ProfileCard(modifier: Modifier = Modifier) {
```
클래스, 함수, 변수 이름에 언더바 사용하는 건 권장되지 않습니다! ProfileCard로 변경해주세요!